lang/test/org/partiql/lang/eval/builtins/windowFunctions/WindowFunctionTests.kt

```diff
@@ -0,0 +1,438 @@
+package org.partiql.lang.eval.builtins.windowFunctions
```

Comment:
Not related to this PR but can you please open an issue in `partiql-tests` to include window function conformance tests (once RFC is approved)?

Reply:
Tracking the issue https://github.com/partiql/partiql-tests/issues/42.